### PR TITLE
 added loggedIn variable to / route if logged in the logout button wi…

### PR DESCRIPTION
**Pull request by Tristenh**
…ll display on main page, removed nav from main and placed in all.handlebars

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -6,39 +6,17 @@
 </head>
 <body>
   <div class="navbar-head">
-  <h1>Your Recipe Book</h1>
+  <h1><a href="/">Your Recipe Book</a></h1>
   <a href="#">Profile</a>
   <a href="#">Contact</a>
          {{#if logged_in}}
-        <a href="/profile">profile</a> |
```
Comment (anUF0) [inline views/layouts/main.handlebars]: Floating token on this line

**Review by anUF0 — COMMENTED** [review]
Floating token on line 13 of all.handlebars

File: views/all.handlebars [hunk 2]
```diff
@@ -1,5 +1,27 @@
+    <nav class="navbar navbar-dark bg-dark navbar-expand-md">
```
Comment (sakshiarora04) [inline views/all.handlebars]: you changed menu code location?

File: views/layouts/main.handlebars (hunk 1 above)
Comment (Tristenh) [reply views/layouts/main.handlebars]: that was deleted

**Review by anUF0 — APPROVED** [review]
LGTM

**Pull request merged**